.github/workflows/test-workflow.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           java-version: ${{ matrix.java }}
       - name: Build and run with Gradle
         working-directory: ${{ env.WORKING_DIR }}
-        run: ./gradlew build ${{ env.BUILD_ARGS }} -x alertingBwcCluster#fullRestartClusterTask -x alertingBwcCluster#mixedClusterTask -x alertingBwcCluster#oldVersionClusterTask0 -x alertingBwcCluster#oldVersionClusterTask1 -x alertingBwcCluster#rollingUpgradeClusterTask -x alertingBwcCluster#twoThirdsUpgradedClusterTask
+        run: ./gradlew assemble integTest ${{ env.BUILD_ARGS }}
```

Comment:
NIT: why do we need to do assemble before running integTest?

Reply:
to ensure it can build the artifact still. We had this in 2.x, so wanted to keep it consistent